src/styles/components/modal/full-screen/styles.less

```diff
@@ -118,10 +118,17 @@
     text-align: center;
   }
 
+  .modal-full-screen-header-with-sub-title {
```

Comment:
I can't find where this class is used?!

Reply:
It's used in `FrameworkConfiguration`...github doesn't open that file because too big, so it probably missed your cmd+f 😏